enterprise/tools/rbeperf/rbeperf.go

```diff
@@ -702,6 +702,12 @@ func (g *runner) run(ctx context.Context) error {
 		remoteOutputUploadHist.Add(remoteStats.uploadOutputsDuration.Milliseconds())
 		remoteTotalHist.Add(remoteStats.totalDuration.Milliseconds())
 
+		if res.RemoteStats.UsageStats == nil {
+			log.Warningf("Missing remote usage stats for %q", res.CommandName)
```

Comment:
This will happen pretty often because we use polling-based stats recording, and the first poll attempt might happen just after the task completes. Not sure how often these are happening but maybe we don't want to treat this as an error